ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -196,6 +196,59 @@ periodics:
           sources:
           - secret:
               name: cluster-secrets-azure
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-azure-key-rotation-e2e
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --git-ref=charlesakalugwu/openshift-azure@key-rotation-again
```

Comment:
Should be updated once https://github.com/openshift/openshift-azure/pull/794 merges.